src/services/events.service.ts

```diff
@@ -0,0 +1,18 @@
+import MyAccountStore from '../app/common/stores/applications/SsvWeb/MyAccount.store';
+import ContractEvent from '~lib/api/ContractEvent';
+
+export const ifEventCaught = async (txHash: string, callBack: Function, delay: Function) => {
```

Comment:
I think this should be in a new file but in some base operator file. Only uses for it are in stores.